FirebaseAI/Sources/Types/Internal/Live/LiveSessionService.swift

```diff
@@ -309,10 +309,9 @@ actor LiveSessionService {
   /// Will apply the required app check and auth headers, as the backend expects them.
   private nonisolated func createWebsocket() async throws -> AsyncWebSocket {
     let host = apiConfig.service.endpoint.rawValue.withoutPrefix("https://")
-    // TODO: (b/448722577) Set a location based on the api config
     let urlString = switch apiConfig.service {
     case .vertexAI:
-      "wss://\(host)/ws/google.firebase.vertexai.v1beta.LlmBidiService/BidiGenerateContent/locations/us-central1"
+      "wss://\(host)/ws/google.firebase.vertexai.v1beta.LlmBidiService/BidiGenerateContent/locations/\(apiConfig.location ?? "us-central1")"
```

Comment:
WDYT about instead adding a location associated value to the VertexAI case of the APIConfig.Service type? That way, the location is couple with the vertexAI case and a nullable location property is not needed.

Reply:
SGTM, done.